### DISCUSSION TITLE
Hashtags in image posts?

### DISCUSSION THREAD

**Discussion by thejaydot** [mastodon/mastodon#33009] [Q&A]
Hi, I have a question about the way hashtags are displayed in image posts. It seems like, if there is no other text in a post, hashtags will only appear in the "hashtag bar" displayed beneath an image, and not in the body of a post. (I gather this bypasses the info [here](https://fedi.tips/what-are-hashtags-how-do-i-use-them-on-mastodon-and-the-fediverse/), which states "for the hashtags to be displayed separately from the main body of the post, you do need to include the blank line break before the list of hashtags.")

I'm wondering if there's any way to choose how hashtags will display in my posts with images – either in the content of the post, or only in the hashtag bar.

Thanks!

**Comment by ghost** [comment]
I was wondering about the position of hashtags too. 

"if there is no other text in a post, hashtags will only appear in the 'hashtag bar' displayed beneath an image". This is not the same in every client it seems. In a browser, yes. But in two iOS apps I checked this is not the case. (app Mastodon and app Ice Cubes).

**Reply by thejaydot** [reply]
Interesting, I was only looking in a browser (Firefox 115.20.0) – some additional consistency with apps would be nice.
